### PR TITLE
documentation updates

### PR DESCRIPTION
**Pull request by nonstoptimm**
- made the readme for verseagility a bit more crisp
- removed scraper docs, as it should not be used
- added basic flair documentation

### CODE REVIEW

File: docs/setup/03 - Project Setup.md [hunk 1]
```diff
@@ -108,6 +108,7 @@ The central components can be found in the script `code/ner.py`.
 Azure Text Analytics is a Cognitive Service providing an API-based extraction of relevant entities from texts. You can find the documentation for Azure Text Analytics API [here](https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/how-tos/text-analytics-how-to-entity-linking?tabs=version-3). In order to use it within the NLP toolkit, you need to [set up a Cognitive Service](https://docs.microsoft.com/en-us/azure/cognitive-services/cognitive-services-apis-create-account?tabs=multiservice%2Cwindows) in your personal Azure subscription and insert the relevant subscription keys. A basic service is free, yet it has request limitations. You can find a description how to set your keys if you scroll up to the top of the page.
 
 #### **Flair Pre-trained NER**
+Flair is a "a powerful NLP library. Flair allows you to apply our state-of-the-art natural language processing (NLP) models to your text, such as named entity recognition (NER), part-of-speech tagging (PoS), special support for biomedical data, sense disambiguation and classification, with support for a rapidly growing number of languages."
```
Comment (TimoA89) [inline docs/setup/03 - Project Setup.md:111]: If that's really a citation, you might want to reference where that citation was taken from

File: docs/setup/03 - Project Setup.md [hunk 2]
```diff
@@ -186,7 +196,11 @@ See the following json-snippet as an example:
         }
 }
 ```
-You see that there are multiple task levels. If you only want to go for classification, keep task level 1 in the config. In case you do not want to integrate Multi Label Classification, Named Entity Recognition, Question/Answering, Opinion Mining..., simply remove it from your JSON.
+You see that there are multiple task levels. To give you a bit of help here, use following examples:
```
Comment (TimoA89) [inline docs/setup/03 - Project Setup.md:199]: Maybe I just don't get it, but why are you talking about "levels" here? I only see task 1, task 2, ... 
Using the terms level, I would expect something like that:
```json
"tasks": {
   "classification": [
      {
            "label": "subcat",
            "type": "classification",
            "model_type": "roberta",
            "max_seq_len": 256,
            "embeds_dropout":0.3,
            "learning_rate":3e-5,
            "prepare": true
        },
        "2": {
            "type": "multi_classification",
            "model_type": "roberta",
            "max_seq_len": 256,
            "embeds_dropout":0.3,
            "learning_rate":2e-5,
            "prepare": true
        }
   ],
   "questionAnswering": []
...
```

File: docs/setup/README.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
-# Verseagility Setup
-This part of the documentation helps you to set up Verseagility in our own subscription and with your data. Each of the points below will lead you to the respective documentation page. At the bottom of each page you will find a link to the previous and next page.
+# Getting Started with Verseagility
+This part of the documentation helps you to set up Verseagility in our own subscription and with your data, step by step. Each of the points below will lead you to the respective documentation page. At the bottom of each page you will find a link to the previous and next page.
 
-1. [Setup](01%20-%20Verseagility%20Setup.md)
+1. [Setup of Code and Infrastructure](01%20-%20Verseagility%20Setup.md)
```
Comment (TimoA89) [inline docs/setup/README.md:4]: Should we also rename the Title of the linked pages to be a little more consistent?

File: docs/setup/03 - Project Setup.md [hunk 4]
```diff
@@ -186,7 +200,46 @@ See the following json-snippet as an example:
         }
 }
 ```
-You see that there are multiple task levels. If you only want to go for classification, keep task level 1 in the config. In case you do not want to integrate Multi Label Classification, Named Entity Recognition, Question/Answering, Opinion Mining..., simply remove it from your JSON.
+You see that there are multiple tasks in the project file. To help you understand how potential changes could look like, use following examples:
+- If you only want to go for classification, keep task level 1 in the config
```
Comment (TimoA89) [inline docs/setup/03 - Project Setup.md:204]: Ich spring von Level zu Level zu Level ;)

**Review by TimoA89 — APPROVED** [review]
